COPYING.md

```diff
@@ -64,24 +49,61 @@ This program is based on works distributed under the terms of another license(s)
     Copyright (c) 2008-2011  David G. Lowe (lowe@cs.ubc.ca). All rights reserved.
     Distributed under the [BSD License](http://www.opensource.org/licenses/bsd-license.php).
 
+*   __geogram__
+    [geogram](http://alice.loria.fr/software/geogram/doc/html/index.html)
+    Copyright (c) 2012-2014 Bruno Levy. All rights reserved.
+    Distributed under the [BSD-3-Clause](https://opensource.org/licenses/BSD-3-Clause)
+
 *   __histogram__
     Copyright (c) Jansson Consulting & Pierre Moulon.
-    Licensed under the [MPL2 license](http://opensource.org/licenses/MPL-2.0).
+    Distributed under the [MPL2 license](http://opensource.org/licenses/MPL-2.0).
 
 *   __htmlDoc__
     Copyright (c) Pierre Moulon.
-    Licensed under the [MPL2 license](http://opensource.org/licenses/MPL-2.0).
+    Distributed under the [MPL2 license](http://opensource.org/licenses/MPL-2.0).
 
 *   __lemon__
     Copyright (c) Lemon authors.
-    Licensed under the [Boost License 1.0](http://www.boost.org/LICENSE_1_0.txt).
+    Distributed under the [Boost License 1.0](http://www.boost.org/LICENSE_1_0.txt).
+
+*   __meshsdfilter__
```

Comment:
so the same holds for MeshSDFilter and OpenMesh.

Also, shall we add CUDA? (also optional)

Reply:
and OpenEXR, but i;m not sure we use it directly?

Reply:
CoinUtils, Clp, Osi (required),
 PopSift CCTag opengv (optional)

Reply:
updated